lib/rules/jsx-sort-props.js

```diff
@@ -333,7 +360,7 @@ module.exports = {
             }
             if (!currentValue && previousValue) {
               report(context, messages.listShorthandFirst, 'listShorthandFirst', {
-                node: memo.name,
+                node: decl.name,
```

Comment:
I think we had an issue here with the `shorthandFirst` node returned in the report. I think that we're expecting the report to be assigned to the shorthand prop and not to the prop that's being compared to. Similar to the report that we do for `reservedFirst`